src/common-priv.h

```diff
@@ -0,0 +1,77 @@
+/*-
+ * Copyright (c) 2019 Ken Cunningham kencu@macports.org
```

Comment:
Need to update this (C) ... either yourself or MP & the new year

Reply:
I intentionally left it as-is, mostly because this is just a refactor/move of code without actual changes.

But yeah, the minor changes in preprocessor conditions could justify an update, so I'll do that.

Reply:
You are creating a new file, which needs a new (C): either totally new or keep the old (C) for the code as copied, but update it for anything new.

Reply:
Updated, with interval to this year.

Reply:
works for me; thx!